src/code/code_800EC960.c

```diff
@@ -143,117 +143,127 @@ u8 sAudioExtraFilter2 = 0;
 Vec3f* sSariaBgmPtr = NULL;
 f32 D_80130650 = 2000.0f;
 u8 sSeqModeInput = 0;
+
+#define SEQ_FLAG_ENEMY (1 << 0)
+#define SEQ_FLAG_FANFARE (1 << 1)
+#define SEQ_FLAG_FANFARE_GANON (1 << 2)
+#define SEQ_FLAG_RESTORE (1 << 3)
```

Comment:
I think just "restore" is unclear about "can this sequence be restored" (it isn't afaict) vs "can a sequence be restored when the sequence is stopped"

Actually I think looking at `func_800F5B58` that if this flag isn't set the sequence isn't even stopped? maybe these can be called "temporary playing sequences" ? :thinking: it's weird that specific functions exist but check this flag nontheless

Reply:
Yeah, it restores but under a specific set of conditions with a specific function, so I don't really know how to describe that with a short name, so I kept it simple and would explain it more with comments later (I intend to properly comment and adjust these flags later, this is indeed a first draft)